SoftLayer/CLI/file/refresh.py

```diff
@@ -9,10 +9,15 @@
 @click.command(cls=SoftLayer.CLI.command.SLCommand, )
 @click.argument('volume_id')
 @click.argument('snapshot_id')
+@click.option('--force-refresh', '-f',
+    help = "Cancel current refresh process and initiates the new refresh.",
+    type = click.BOOL,
+    default = False,
+    show_default = True)
```

Comment:
Use this style, here and in the other command.
```suggestion
@click.option('--force-refresh', '-f', is_flag = True, default = False, show_default = True,
              help = "Cancel current refresh process and initiates the new refresh."
)
```

Reply:
Sure, will look into suggestions and fix indentation as well